tests/test_panel_daily_bar_reader.py

```diff
@@ -13,34 +13,36 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from itertools import permutations
+
 import pandas as pd
 
 from zipline.data.us_equity_pricing import PanelDailyBarReader
-from zipline.testing.fixtures import WithTradingEnvironment, ZiplineTestCase
+from zipline.testing import ExplodingObject
+from zipline.testing.fixtures import ZiplineTestCase
 
 
-class TestPanelDailyBarReader(WithTradingEnvironment, ZiplineTestCase):
+class TestPanelDailyBarReader(ZiplineTestCase):
     def test_duplicate_values(self):
-        df = pd.DataFrame()
-        panel = pd.concat([pd.Panel({"X": df}), pd.Panel({"X": df})])
-
-        with self.assertRaises(ValueError) as e:
-            # panel's items has duplicates
-            PanelDailyBarReader(None, panel)
-
-        self.assertEqual("Duplicated items found: ['X']",
```

Comment:
I wanted to include the axis names in the message, which meant I had to update these entries, and it seemed simpler to just calculated the expected name from the transpose, which made it easer to just iterate over all possible transpositions.